### DISCUSSION TITLE
ISS trail needs special handling - by design?

### DISCUSSION THREAD

**Discussion by hn-88** [OpenSpace/OpenSpace#3187] [General]
At present, (v19.2), with the default profile, pressing 'H' hides all planet and satellite trails except ISS trail. This seems to be because the ISS trail property has a lowercase 'trail' while the other trail property names have an uppercase 'Trail'. Is this by design? If this is deliberate and ISS trail should not be hidden along with other satellite / planet trails, can we assign a hotkey to hiding it in the default profile? instead of having to manually create a new profile just to hide the ISS trail?

**Comment by alexanderbock** [comment]
1. Yes, it is intentional.  In `data/assets/base_keybindings.asset` we bind the `h` key to the "toggle_trails_planets_moons" action, so only trails of moons and planets should be hidden
2. For the upcoming 0.20.0 version, we have (a) changed `h` to `t` to better map that we are toggling *T*rails and also change it that `t` now toggles _all_ trails (https://github.com/OpenSpace/OpenSpace/blob/master/data/assets/base_keybindings.asset#L25-L26)

If you want to permanently fix it on your installation, you could edit the base_keybindings.asset in your installation.  In general we don't recommend it, as it is easy to forget when you are updating to a new OpenSpace version later and will have to redo it.  Creating a new profile for yourself is the more "safe" way

**Reply by hn-88** [reply]
Thanks @alexanderbock - Happy to see the enhancement planned for 0.20.

**Reply by alexanderbock** [reply]
Yea, the Aqua and Terra satellites had a small enough field of view, that they would miss parts of the Earth on each pass around :/

Check if ESA's Copernicus satellites have the correct data: https://browser.dataspace.copernicus.eu
